modules/internal/ChapelArray.chpl

```diff
@@ -2636,6 +2638,13 @@ module ChapelArray {
       }
     }
 
+    pragma "no doc"
```

Comment:
Separately, or as part of the main `push_front` / `push_back` documentation?

Reply:
I lean toward separately since I see them as having different signatures vs. just being promoted versions of the single element ones.  Either way could work for me though.